easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1858,83 +1860,72 @@ def det_installversion(version, toolchain_name, toolchain_version, prefix, suffi
     _log.nosupport('Use det_full_ec_version from easybuild.tools.module_generator instead of %s' % old_fn, '2.0')
 
 
-def get_easyblock_class(easyblock, name=None, error_on_failed_import=True, error_on_missing_easyblock=True, **kwargs):
```

Comment:
Don't remove `error_on_failed_import`, that's API breakage.

At best deprecate it

Reply:
Well, we can actually break the API between major versions :wink: I re-added it with a deprecation warning nonetheless. Done in https://github.com/easybuilders/easybuild-framework/pull/4531/commits/eacd6e3b65f38c2b0505a2e5f873ad1eae72cc1a